main.py

```diff
@@ -37,21 +37,22 @@ def reEmail(fname):
 def reAward(fname):
   fhand = open(fname, 'r')
   for line in fhand:
-    extr = re.findall('.*', line)
+    extr = re.findall('-.+', line)
```

Comment:
Use brackets to EXTRACT a portion of the match.

`extr = re.findall('- (.*)', line)`